runtime/src/iree/task/queue_test.cc

```diff
@@ -6,8 +6,26 @@
 
 #include "iree/task/queue.h"
 
+#include <thread>
+
 #include "iree/testing/gtest.h"
 
+// Like iree_task_queue_try_steal but retries until success.
+// This is used in this test as iree_task_queue_try_steal may (rarely) fail even
+// in simple tests that don't look particularly contended, see #15488.
```

Comment:
```suggestion
// in simple tests with no contention, see #15488.
```

Or perhaps even "with no concurrency". The tests use queues allocated on the stack and there is no threading involve in the individual testcases so there is absolutely no concurrency on the mutex. So let's make it clear that it can fail even though we know the lock is free (e.g. on AArch64 due to interrupt, cache eviction or the like that causes the exclusive monitor to be dropped before the stxr).

Reply:
done!